internal/suggest/testData/nameConflict.yaml

```diff
@@ -0,0 +1,192 @@
+openapi: 3.1.0
```

Comment:
Nit: dir name should be `testdata`

Reply:
I understand that's a go recommendation, but I'm vehemently against that naming convention